src/intentLogic/getHeadlines.js

```diff
@@ -5,31 +5,40 @@ const helpers = require('../helpers');
 const speech = require('../speech').speech;
 const sound = require('../speech').sound;
 
-module.exports = function () {
-	this.event.session.attributes.lastIntent = 'GetHeadlines';
-	this.event.session.attributes.positionalContent = [];
+module.exports = function (offSet) {
```

Comment:
As far as I can tell you're using the offset passed in here as a flag, to tell whether or not a new intent has been called. I find it confusing to have this offset and another offset in the context - really you only need the one in the context, since 3 is hardcoded below.
Or am I missing something?